link/src/main/java/com/stripe/android/link/ui/verification/VerificationDialog.kt

```diff
@@ -28,18 +28,12 @@ import com.stripe.android.link.theme.linkShapes
 import com.stripe.android.link.ui.LinkAppBar
 import com.stripe.android.link.ui.rememberLinkAppBarState
 
-/**
- * Function called when the Link verification dialog has been dismissed. The boolean returned
- * indicates whether the verification succeeded.
- * When called, [LinkPaymentLauncher.accountStatus] will contain the up to date account status.
- */
-typealias LinkVerificationCallback = (success: Boolean) -> Unit
```

Comment:
This looks like a source breaking change. (Consumers can use this public type)

Reply:
True. While they’d only have access if they depend directly on `com.stripe:link` (which I don’t know that we should actually expose), I can mark this with an `@Deprecated` annotation instead for now. Sound good?

Reply:
Sounds good to me.